appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
Yea, sorry for that confusion, I've been using "multi-generic event" and "multi-event" pretty interchangeably.

Also, yea, I created a new Yail "define-multi-event" for those blocks (still in the same function that creates the Yail for normal events and uses "define-event"). And then, I pretty much tried to mimic everything that happens for define-event, for define-multi-event, which doesn't happen an instance associated with it, just a the component type.